dref/enums.py

```diff
@@ -0,0 +1,91 @@
+import enum
```

Comment:
Let's add a small note telling why we are defining this here, what is the source and when we can remove this (after upgrading Django to 3)